README.md

```diff
@@ -1,2 +1,209 @@
-# module-ballerinax-zendesk
-The Ballerina Zendesk Connector facilitates direct integration and automation between Ballerina applications and the Zendesk platform. This connector is designed for developers requiring efficient integration solutions to extend customer support functionalities within Ballerina applications, minimizing complexity in API communication.
+# Ballerina Zendesk connector
+
+[![Build](https://github.com/ballerina-platform/module-ballerinax-zendesk/actions/workflows/ci.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-zendesk/actions/workflows/ci.yml)
+[![Trivy](https://github.com/ballerina-platform/module-ballerinax-zendesk/actions/workflows/trivy-scan.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-zendesk/actions/workflows/trivy-scan.yml)
+[![codecov](https://codecov.io/gh/ballerina-platform/module-ballerinax-zendesk/branch/master/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerinax-zendesk)
+[![GraalVM Check](https://github.com/ballerina-platform/module-ballerinax-zendesk/actions/workflows/build-with-bal-test-graalvm.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-zendesk/actions/workflows/build-with-bal-test-graalvm.yml)
+[![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerinax-zendesk.svg)](https://github.com/ballerina-platform/module-ballerinax-zendesk/commits/master)
+[![GitHub Issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-library/module/github.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-library/labels/module%2Fgithub)
+
+Zendesk is a customer service software company that provides a cloud-based customer support platform. It is designed to
+offer a seamless and efficient customer service experience, enabling businesses to manage customer interactions across
+multiple channels, including email, chat, phone, and social media.
```

Comment:
As a best practice, shall we remove the new lines within paragraphs? There are some discrepancies in various MarkDown renders. Therefore, having new lines inside paragraphs might cause issues in some renderers.

Reply:
+1. Addressed in https://github.com/ballerina-platform/module-ballerinax-zendesk/pull/3/commits/b7f2b18f72493a877a10abd3b9b2c6b58aa971be